model/src/main/kotlin/Package.kt

```diff
@@ -147,10 +147,8 @@ data class Package(
         val EMPTY = Package(
             id = Identifier.EMPTY,
             purl = "",
-            authors = emptySet(),
```

Comment:
Actually, I believe that by convention we should always set all arguments for any `val EMPTY`. Because if ever a default value changes to something non-"empty", then you don't need to remember to adjust that parameter here as well.

Some places already do set all values despite their default values being empty / null (like `OrtResult`), other's don't (like `ScanSummary`). IMO we should align *all* of these to set *all* values.

Reply:
I've dropped the commit for now.